cmd/root.go

```diff
@@ -176,39 +64,21 @@ func getGitHubClient(token string) (*github.Client, error) {
 }
 
 // getJIRAClient initializes a JIRA API client, then sets the Basic Auth credentials
-// passed to it. (OAuth token support is planned.) It then requests the project using
-// the key provided on the command line, to have it accessible by future functions and
-// to check that the API is accessible and the auth credentials are valid.
-func getJIRAClient(username, password, baseURL string) (*jira.Client, error) {
-	client, err := jira.NewClient(nil, baseURL)
-	if err != nil {
-		log.Errorf("Error initializing JIRA client; check your base URI. Error: %v", err)
-		return nil, err
-	}
-	client.Authentication.SetBasicAuth(username, password)
-
-	log.Debug("JIRA client initialized; getting project")
+// passed to it. (OAuth token support is planned.)
+//
+// The validity of the client and its authentication are not checked here. One way
+// to check them would be to call config.LoadJIRAConfig() after this function.
+func getJIRAClient(config lib.Config) (*jira.Client, error) {
```

Comment:
this should be part of the the clients file. lib should know how to create a new jira client given a config.